kvmagent/kvmagent/plugins/imagestore.py

```
@@ -173,9 +173,16 @@ def _parse_json_and_update_mode(volume_infos, json_data):
                 json_data = fd.read()
                 return  _parse_json_and_update_mode(volume_infos, json_data)
 
-    def stop_vm_cbt_backup_jobs(self, vm, force=False):
+    def stop_vm_cbt_backup_jobs(self, vm, records, force=False):
+        infos = ""
+        bitmapName = ""
+        for record in records:
+            infos += ",".join([record.scratchNodeName, record.target]) + ";"
+            if record.lastBitmapName:
+                bitmapName = record.lastBitmapName
         with linux.ShowLibvirtErrorOnException(vm):
```

Ruff (0.8.2)

`linux` may be undefined, or defined from star imports

(F405)